### ISSUE TITLE
Question: how to get interface admin status as up by default

### ISSUE THREAD

**Issue by mdivyamohan** [Azure/sonic-swss#988]
I need an interface to be in admin status up by default (during startup).
Currently interface does not come in admin up state unless I execute 'config interface startup <iface>' command.
What is the correct way to achieve this? Should admin status be set as up in config_db.json or ports.json file?

**Comment by prsunny** [comment]
You can set as "up" in config_db.json

**Comment by mdivyamohan** [comment]
@prsunny Thanks for the clarification.

One more doubt. After startup, later on, how to change the admin status of a port?

I am able to do this with 'config interface..' CLI command. The CLI command seems to be directly acting on CONFIG_DB.

But, is it possible to make this change with json configuration? 
I tried with swssconfig with no success. Is it not possible with swssconfig, as it acts on APPL_DB?

**Comment by prsunny** [comment]
I don't think you can. Not sure what is your intention here. For changing admin up/down, Sonic has the CLI support. This modifies the config_db. You cannot set admin status by changing APPL_DB and swssconfig acts on APPL_DB.

**Comment by stcheng** [comment]
@mdivyamohan yes you could use JSON file to execute the modification; however, it's the same as using the CLI; once you have the configuration, you could use `sonic-cfggen` command to write to the configuration database. you shall not manipulate the application database by any means
